config/peerpods/podvm/Dockerfile.podvm-builder

```diff
@@ -24,8 +24,9 @@ ARG ACTIVATION_KEY
 
 RUN mkdir -p /scripts
 
-ADD lib.sh libvirt-podvm-image-handler.sh aws-podvm-image-handler.sh azure-podvm-image-handler.sh /scripts/
+ADD lib.sh libvirt-podvm-image-handler.sh aws-podvm-image-handler.sh azure-podvm-image-handler.sh gcp-podvm-image-handler.sh /scripts/
 
+# TODO: This means we are installing the azure rpms for all providers?
```

Comment:
no, by following it you'll see it's actually installing general set of binaries, i'm not sure it's implemented like, it may need a fix

Reply:
Yes, I just noticed the basic packages. That said, calling an azure specific script here feels a bit confusing.

Maybe renaming the script or moving that installation to another one could help? Anyway, thanks for the clarification! I'll remove the note and close this thread since it can be handled in a different PR.